db/migrate/20160329231848_text_search_indices.rb

```diff
@@ -0,0 +1,12 @@
+class TextSearchIndices < ActiveRecord::Migration
+  def change
+    DbTextSearch::CaseInsensitiveEq.add_index connection, :thredded_categories, :name, name: :thredded_categories_name_ci
+    # Recreate previously existing MySQL index in favour of consistent names
```

Comment:
Instead of the comment here what do you think about extracting the conditional and its contents into some descriptive methods like ...

```ruby
remove_old_indexes if mysql_adapter?
```